swagger.yaml

```diff
@@ -312,6 +314,12 @@ definitions:
       href:
         type: string
         description: "The URL to the resource"
+  Href:
```

Comment:
Because this is a generic object, you are forced to use a generic description - which doesn't seem correct as a user (or in this case me) may ask url to what resource? See the dataset API swagger specs and the use of descriptions to describe each unique href.